app/templates/index.html

```diff
@@ -8,7 +8,6 @@
 <div id="controls">
     <input type="text" id="query" placeholder="Search">
     <button id="submit"><i class="fa fa-search"></i></button>
-    <div class="filters">
         <button id="clear_filters">
```

Comment:
Have you been able to test this? Unfortunately I think this would break the filters quite severely. There still needs to be a filters div wrapping them, it just needs to be shown in the same row as the other controls—this will probably be a CSS change.